doc/architecture.txt

```diff
@@ -28,5 +28,5 @@ The MQTT library included in this SDK is a modification of the [Paho C MQTT clie
 
 @subsection architecture_shadow Thing Shadow
 
-This SDK provides a client implementation for AWS IoT Thing Shadows. Thing Shadows use a JSON-based protocol that lets applications communicate with devices even when the device is not connected.
+This SDK provides a client implementation for AWS IoT Thing Shadows. A device's Shadow is a JSON document that is used to store and retrieve its current state information regardless of whether it is connected to the Internet.
```

Comment:
* The "..used to store and retrieve its current information.." seems incomplete about where it will be stored and what will retrieve the current state of the device (when it is not connected to the internet
* I think linking to the Shadow AWS docs would be helpful.

Reply:
Fixed